lib/main.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
     // convert Buffers before splitting into lines and processing
     src.toString().split('\n').forEach(function (line) {
       // matching "KEY' and 'VAL' in 'KEY=VAL'
-      var keyValueArr = line.match(/^\s*([\w\.\-]+)\s*=\s*(.*)?\s*$/)
+      var keyValueArr = line.match(/^\s*(?:export)?\s*([\w\.\-]+)\s*=\s*(.*)?\s*$/)
```

Comment:
The non-capture group needs to include the `\s` and you need to replace the `*` after it with `+`:

```js
var keyValueArr = line.match(/^\s*(?:export\s+)?([\w\.\-]+)\s*=\s*(.*)?\s*$/)
```

Otherwise you'll end up matching `exportedKey=foo` as `['exportedKey=foo', 'edKey', 'foo']`